packages/uniforms-material/src/ErrorField.js

```diff
@@ -1,19 +1,21 @@
-import ErrorOutline   from 'material-ui/svg-icons/alert/error-outline';
-import React          from 'react';
 import connectField   from 'uniforms/connectField';
 import filterDOMProps from 'uniforms/filterDOMProps';
+import FormControl    from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
 import nothing        from 'uniforms/nothing';
-import {ListItem}     from 'material-ui/List';
+import React          from 'react';
 
-const Error = ({children, error, errorMessage, ...props}) =>
+const Error = ({children, error, errorMessage, fullWidth, margin, ...props}) =>
     !error ? nothing : (
-        <ListItem
-            disabled
-            leftIcon={<ErrorOutline />}
-            primaryText={children || errorMessage}
-            {...filterDOMProps(props)}
-        />
+        <FormControl error={!!error} fullWidth={!!fullWidth} margin={margin}>
+            <FormHelperText {...filterDOMProps(props)}>{children || errorMessage}</FormHelperText>
+        </FormControl>
```

Comment:
Are you sure about the `props` being passed to the `FormHelperText`? Maybe we should pass them to the `FormControl` instead?

Same goes for `ErrorsField`. And other `FormControl` usages too.